tools/docker/tests/test_flows.sh

```diff
@@ -151,6 +156,8 @@ test_init() {
         exit 1
     }
     AL_MAC=$(docker exec -it gateway ${installdir}/bin/beerocks_cli -c bml_conn_map | grep IRE_BRIDGE | awk '{print $5}' | cut -d ',' -f 1)
+    RADIO_WLAN0_MAC=$(docker exec -it gateway ${installdir}/bin/beerocks_cli -c bml_conn_map | grep "RADIO: wlan0" | head -1 | awk '{print $4}' | cut -d ',' -f 1)
```

Comment:
Add also `RADIO_WLAN2_MAC`. Might not be used for now, but can be useful for the future.